x/evm/client/rest/query.go

```diff
@@ -59,12 +59,15 @@ func GetHandlerQueryPendingCommands(cliCtx client.Context) http.HandlerFunc {
 		}
 
 		chain := mux.Vars(r)[utils.PathVarChain]
-		res, err := evmclient.QueryPendingCommands(cliCtx, chain)
+
```

Comment:
I would completely get rid of this rest handler, you already defined the new rest endpoint in the proto file

Reply:
rest handler removed